package.json

```diff
@@ -196,6 +197,11 @@
                 }
             ],
             "view/title": [
+                {
+                    "command": "aws.submitFeedback",
+                    "when": "view == aws.explorer",
+                    "group": "navigation@6"
```

Comment:
Does this still have the same order as the screenshot?

Reply:
Yes, just incremented it to ensure it's to the riht